### PR TITLE
Replace phantomjs with headless chrome

### PR DESCRIPTION
**Pull request by Dantemss**
Hopefully less flaky and closer to chrome's behavior? You can also change the rails_helper line to `Capybara.javascript_driver = :chrome` to see the tests in progress on an actual chrome window.

### CODE REVIEW

**Comment by Dantemss** [comment]
I plan to run this build a few times and see if I can get the feature specs to fail like they normally do.

File: spec/features/admin/assign_teachers_spec.rb [hunk 1]
```diff
@@ -49,6 +48,7 @@
 
     # Remove the teacher
     click_link 'Remove from course'
+    page.driver.browser.switch_to.alert.accept
```
Comment (jpslav) [inline spec/features/admin/assign_teachers_spec.rb]: Can this be in a nicely-named helper method in `rails_helper`?

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -40,19 +40,30 @@
...
 
-require 'capybara'
-require 'capybara/poltergeist'
-Capybara.javascript_driver = :poltergeist
-window_size = [1920, 6000]
+require 'selenium/webdriver'
 
-Capybara.asset_host = 'http://localhost:3001'
-
-require 'screenshots'
```
Comment (jpslav) [inline spec/rails_helper.rb]: Is this gone now?

Comment (Dantemss) [reply spec/rails_helper.rb]: Moved to spec/support to it's not included in code coverage calculations

File: spec/features/admin/assign_teachers_spec.rb (hunk 1 above)
Comment (Dantemss) [reply spec/features/admin/assign_teachers_spec.rb]: Sure

Comment (Dantemss) [reply spec/features/admin/assign_teachers_spec.rb]: Added `alert` method to `spec/feature_js_helper.rb`

**Pull request merged**